vault/datadog_checks/vault/metrics.py

```diff
@@ -121,6 +121,7 @@
     'vault_replication_merkleDiff': 'replication.merkleDiff',
     'vault_replication_merkleSync': 'replication.merkleSync',
     'vault_replication_merkle_commit_index': 'replication.merkle.commit_index',
+    'vault_replication_wal_gc': 'replication.wal.gc',
```

Comment:
Are we able to get this metric in tests?

Reply:
Unfortunately no